frontend/src/metabase/query_builder/components/filters/DateOperatorFooter.tsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable react/prop-types */
```

Comment:
You can remove `eslint-disable` here

Reply:
Eslint complained here but I removed it where I could